workspace-extension/src/extension.ts

```diff
@@ -22,25 +22,92 @@ function parsePort(input: string): number | string {
     parsed < MIN_PORT_INCLUSIVE ||
     parsed >= MAX_PORT_EXCLUSIVE
   ) {
-    return `Port number must be an integer between ${MIN_PORT_INCLUSIVE} and ${MAX_PORT_EXCLUSIVE}`;
+    return `Port number must be an integer between ${MIN_PORT_INCLUSIVE} and ${MAX_PORT_EXCLUSIVE} but was "${input}"`;
   }
   return parsed;
 }
 
-function parseTarget(input: string): Hostname | string {
+/**
+ * Port expressions can be:
+ *   A list of ports eg. 2000,2010,2020
+ *   A range of ports eg. 2000-2020
+ *   A start port and a number of ports eg. 2000x2
```

Comment:
@Danielv123 , i decided to support multiple syntaxes for ports, I decided to use `AxB` for the portcount style you were relying on because commas made more sense for the portlist format IMO.

Reply:
Sweet, thanks!